app/index.ejs

```diff
@@ -6,13 +6,10 @@
   <meta charset="utf-8">
   <title>Cosmos Voyager</title>
   <% if (htmlWebpackPlugin.options.appModules) { %>
-    <!-- Add `app/node_modules` to global paths so `require` works properly in development -->
-    <script>
-      require('module').globalPaths.push('<%= htmlWebpackPlugin.options.appModules.replace(/\\/g, '\\\\') %>')
-    </script>
     <% } %>
 
       <body style="background: #15182d">
+        <script async src='https://www.google-analytics.com/analytics.js'></script>
```

Comment:
can we package this with the app instead of making this call?

Reply:
Sure.